zerver/lib/queue.py

```diff
@@ -187,7 +187,7 @@ def _connect(self, on_open_cb: Optional[Callable[[], None]]=None) -> None:
         self.connection = ExceptionFreeTornadoConnection(
             self._get_parameters(),
             on_open_callback = self._on_open,
-            stop_ioloop_on_close = False)
+        )
```

Comment:
Well, we originally had a , not a 

I'd blame this part of the pika documentation for `pika.adapters.TornadoConnection`:
```
    """The TornadoConnection runs on the Tornado IOLoop. If you're running the
    connection in a web app, make sure you set stop_ioloop_on_close to False,
    which is the default behavior for this adapter, otherwise the web app
    will stop taking requests.
```

Reply:
Ah, indeed not the best prose -- "set" is the wrong verb there.

The relevant information is right there after it in that sentence, though ("which is the default behavior for this adapter"), in addition to the function signature above, so it was only unclear, not wrong.